tests/server/helpers.js

```diff
@@ -24,3 +25,14 @@ export function checkSRI(res) {
       'anonymous', 'script should have crossorigin attr');
   });
 }
+
+export function parseCSP(rawCsp) {
+  return camelizeKeys(
```

Comment:
Nitpick but could we import `camelizeKeys` as `camelCaseKeys`? It feels like a more descriptive function name and camelize isn't a real word.

If you don't want to just ignore this comment, it's fine.